packages/calcite-components/src/components/switch/switch.scss

```
@@ -4,11 +4,12 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-switch-background-color: Specifies the component's background color.
- * @prop --calcite-switch-border-color: Specifies the component's border color.
+ * @prop --calcite-switch-background-color-hover: Specifies the component's background color when hovered or pressed.
+ * @prop --calcite-switch-border-color: [Deprecated] Specifies the component's border color.
```

Should we specify any deprecation reasoning or what they should use instead?

---

We don't have other similar references across our CSS doc, but since the border is no longer applicable, maybe we could go with:

```suggestion
 * @prop --calcite-switch-border-color: [Deprecated] No longer necessary. Specifies the component's border color.
```

cc @DitwanP